data/json/recipes/chem/other.json

```
@@ -109,7 +109,8 @@
     "batch_time_factors": [ 83, 5 ],
     "autolearn": true,
     "charges": 10,
-    "book_learn": [ [ "recipe_arrows", 1 ], [ "textbook_survival", 1 ], [ "survival_book", 1 ] ],
+    "//": "Veterinary handbook teaching this recipe is more of an easter egg but is intentional",
+    "book_learn": [ [ "recipe_arrows", 1 ], [ "textbook_survival", 1 ], [ "survival_book", 1 ], [ "reference_firstaid2", 4 ] ],
```

```suggestion
    "book_learn": [ [ "recipe_arrows", 1 ], [ "textbook_survival", 1 ], [ "survival_book", 1 ], [ "reference_firstaid2", 2 ] ],
```

is lvl 4 even as an easter egg a bit too steep?